docker/docker-compose-init-db.yml

```
@@ -0,0 +1,38 @@
+version: '3.2'
+services:
+
+  # ICUBAM service
+  db-builder:
+    build:
+      context: "${ICUBAM_COMPOSE_CONTEXT:-..}"
```

See comment below about the mount
```suggestion
      context: "${ICUBAM_COMPOSE_CONTEXT:-..}"
```

---

Ok, I accepted the changes